### DISCUSSION TITLE
"Configure new cluster" button absent when starting Kafka UI from jar

### DISCUSSION THREAD

**Discussion by mstezhka** [provectus/kafka-ui#4339] [Q&A] [answered]
Hi team,
When I running Kafka UI app from the jar this button is not shown in UI.
When Kafka UI app is started from docker image - it's good.
I did try from version 0.3.3 to the latest one - the same issue for all of them.
Is it a bug, or some workaround exists?

**Comment by Haarolean** [comment, marked as answer]
you're missing the dynamic config property, see here: [docs](https://docs.kafka-ui.provectus.io/configuration/configuration-wizard)
